kfdef/kfctl_openshift.yaml

```diff
@@ -51,7 +45,7 @@ spec:
       repoRef:
         name: manifests
         path: kafka/kafka
-    name: kafka-cluster
+    name: kafka-cluster 
```

Comment:
looks like maybe an unnecessary extra space showed up here

Reply:
deleted it